src/StringTools/StringTools.csproj

```diff
@@ -6,7 +6,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <IsPackable>true</IsPackable>
     <GenerateReferenceAssemblySource>true</GenerateReferenceAssemblySource>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>11.0</LangVersion>
```

Comment:
Should this be dropped or `latest`?

Reply:
If it's just there temporarily to work around the compiler bug we can leave it, but I don't know of a great reason to explicitly specify an old langver . . .

Reply:
11 was the workaround because that's what part of the error message gave, but we can move this to latest and never worry about it again (🤞)